move/lesson/sources/ride.move

```diff
@@ -44,6 +44,23 @@ module suitaxi::ride {
         initial_location: String
     }
 
+    struct RideAcceptedEvent has drop, copy {
```

Comment:
Make sure that the flow in Move file is:

std uses
sui uses
consts
structs 
events
init()
business functions
helper functions